README.md

```diff
@@ -62,13 +62,13 @@ curl https://risingwave.com/sh | sh
 To learn about other installation options, such as using a Docker image, see [Quick Start](https://docs.risingwave.com/docs/current/get-started/).
 
 ## When is RisingWave the perfect fit?
-RisingWave is the ideal solution for building event-driven applications. Choose RisingWave when you want to:
+RisingWave is the ideal solution for:
 
-* Ingest data from real-time sources like Kafka streams, database CDC, and more.
-* Perform complex queries (such as joins, aggregations, and time windowing) on the fly.
+* Handling real-time data sources like Kafka streams, database CDC, and more.
+* Performing complex, on-the-fly queries such as joins, aggregations, and time windowing.
 * Interactively and concurrently explore consistent, up-to-the-moment results.
 * Seamlessly send results to downstream systems.
```

Comment:
```suggestion
* Interactively and concurrently exploring consistent, up-to-the-moment results.
* Seamlessly sending results to downstream systems.
```

should we use ing here @WanYixian ?

Reply:
We can use -ing here because these bullet points are kind of the latter part of the overall sentence: 
"RisingWave is the ideal solution for..."